nvdaHelper/vbufBackends/gecko_ia2/gecko_ia2.cpp

```diff
@@ -532,12 +541,9 @@ VBufStorage_fieldNode_t* GeckoVBufBackend_t::fillVBuf(
 	if(pacc->get_accName(varChild,&name)!=S_OK)
 		name=NULL;
 
-	wstring description;
-	BSTR rawDesc=NULL;
-	if(pacc->get_accDescription(varChild,&rawDesc)==S_OK) {
-		description=rawDesc;
-		parentNode->addAttribute(L"description",description);
-		SysFreeString(rawDesc);
+	std::optional<wstring> description{ getAccDescription(pacc, varChild) }; // todo: fix usages to use optional

```

Comment:
Is this comment still valid? What do you mean exactly?

Reply:
No, sorry. It was just a reminder to myself. I'll remove.